src/test/clojure/clara/test_rules.clj

```diff
@@ -5452,5 +5452,25 @@
                     (insert (->Cold -10))
                     (fire-rules {:cancelling true})
                     (query cold-query))))))
-    
-  
+
+;; Test for https://github.com/cerner/clara-rules/issues/267
+(deftest local-scope-visible-in-join-filter
```

Comment:
Nitpick: Almost all of the other tests in the file have names prefixed by "test" so we should probably make this "test-local-scope-visible-in-join-filter".